### PR TITLE
Add subscription field

### PR DESCRIPTION
**Pull request by ChaituVR**
To keep track of subscirptions

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -414,7 +414,8 @@
       "url": "https://www.oklink.com/manta"
     },
     "start": 332890,
-    "logo": "ipfs://bafkreibbrwgwdcfh755dqh4ndta6tja77qbaa36bfv4pstub6prhhore7a"
+    "logo": "ipfs://bafkreibbrwgwdcfh755dqh4ndta6tja77qbaa36bfv4pstub6prhhore7a",
+    "subscription": "Feb 26, 2024"
```
Comment (ChaituVR) [inline src/networks.json:418]:
```suggestion
    "subscription": "Feb 26, 2025"
```

Comment (wa0x6e) [reply src/networks.json]: Why no timestamp ?

Comment (ChaituVR) [reply src/networks.json]: I thought it should be more human-readable, and if we do `new Date("Feb 26, 2025")` it will return a timestamp

**Comment by bonustrack** [comment]
@ChaituVR We don't need this, we keep track of this in Notion with others infos

**Pull request merged**